src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -130,7 +130,13 @@ public enum ErrorCode {
   VECTORIZE_SERVICE_TYPE_UNAVAILABLE("Vectorize service unavailable : "),
   VECTORIZE_USAGE_ERROR("Vectorize search can't be used with other sort clause"),
 
-  VECTORIZECONFIG_CHECK_FAIL("Internal server error: VectorizeConfig check fail");
+  VECTORIZECONFIG_CHECK_FAIL("Internal server error: VectorizeConfig check fail"),
+
+  UNAUTHORIZED_REQUEST("UNAUTHENTICATED: Invalid token"),
```

Comment:
Wrong enum: "UNAUTHORIZED" should be "UNAUTHENTICATED"?

Reply:
Thanks